tools: Missing implementation for `block_search` in rpc_probe

When introducing proper checks for the `tools` workspace in #997 said checks fail with:

```
error[E0425]: cannot find function `block_search` in this scope
  --> rpc-probe/src/quick.rs:24:17
   |
24 |                 block_search("block.height > 1", 1, 10, "asc").with_name("block_search"),
   |                 ^^^^^^^^^^^^ not found in this scope
```

**Steps to reproduce**

From the project root run: `cargo build-tools`

**What's the definition of "done" for this issue?**

`cargo build-tools` runs without error.

--- a/tools/rpc-probe/Makefile.toml
+++ b/tools/rpc-probe/Makefile.toml
@@ -1,6 +1,6 @@
 [env]
 CONTAINER_NAME = "kvstore-rpc-probe"
-DOCKER_IMAGE = "informaldev/tendermint:0.34.9"
+DOCKER_IMAGE = "informaldev/tendermint:0.34.13"
 HOST_RPC_PORT = 26657
 CARGO_MAKE_WAIT_MILLISECONDS = 3500
 

--- a/tools/rpc-probe/src/kvstore.rs
+++ b/tools/rpc-probe/src/kvstore.rs
@@ -30,6 +30,19 @@ pub fn block(height: u64) -> PlannedInteraction {
     .into()
 }
 
+pub fn block_search(query: &str, page: u32, per_page: u32, order_by: &str) -> PlannedInteraction {
+    Request::new(
+        "block_search",
+        json!({
+            "query": query,
+            "page": format!("{}", page),
+            "per_page": format!("{}", per_page),
+            "order_by": order_by,
+        }),
+    )
+    .into()
+}
+
 pub fn block_results(height: u64) -> PlannedInteraction {
     Request::new(
         "block_results",

